cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/node/CloudNetBridgeModule.java

```diff
@@ -85,40 +100,40 @@ public ProxyFallbackConfiguration createDefaultFallbackConfiguration(String targ
     }
 
     public void writeConfiguration(BridgeConfiguration bridgeConfiguration) {
-        this.getConfig().append("config", bridgeConfiguration);
-        this.saveConfig();
+        getConfig().append("config", bridgeConfiguration);
+        saveConfig();
```

Comment:
Why do you remove the this accessors?

Reply:
I've already wrote a message about this problem. I started editing the master branch 😬 and then moved to the developement branch. I shouldn't use copy and paste xD